account_invoice_refund_line_selection/tests/test_refund_line.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 Creu Blanca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestInvoiceRefundLine(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
+        cls.in_invoice = cls.init_invoice(
+            "in_invoice", products=cls.product_a + cls.product_b
+        )
+        cls.in_invoice._post()
+        cls.out_invoice = cls.init_invoice(
+            "out_invoice", products=cls.product_a + cls.product_b
+        )
+        cls.out_invoice._post()
+
+    def test_partial_refund_in_invoice(self):
+        reversal = (
```

Comment:
I would test with a Form, because there usually exist some on-changes on the wizards.